pyproject.toml

```
@@ -69,7 +69,7 @@ docs = [
     "myst-parser",
     "pydocstyle",
     "sphinx-autodoc-typehints<1.23",
-    "sphinx-rtd-theme",
+    "sphinx-rtd-theme==2.0.0",
```

```suggestion
    "sphinx-rtd-theme<3.0.0",
```

Suggest an upper bound over an absolute value